src/SFML/Graphics/Text.cpp

```diff
@@ -231,6 +294,11 @@ void Text::draw(RenderTarget& target, RenderStates states) const
 
         states.transform *= getTransform();
         states.texture = &m_font->getTexture(m_characterSize);
+
+        // Only draw the outline if there is something to draw
+        if (m_outlineThickness > 0)
```

Comment:
For reference, [```sf::Shape```](https://github.com/SFML/SFML/blob/master/src/SFML/Graphics/Shape.cpp#L222) contains the only other check for outline thickness and it uses a ```!=``` comparison. I can change the implementation here to match that if I understand correctly that nobody wants to see ```if (m_outlineThickness)```.